tests/ballerina-compiler-api-test/src/test/java/io/ballerina/semantic/api/test/statements/SymbolsInMatchStmtTest.java

```diff
@@ -112,7 +112,7 @@ public void testType(int sLine, int sCol, int eLine, int eCol, TypeDescKind type
                 {20, 10, 20, 13, TypeDescKind.ANY},
                 {22, 20, 22, 23, TypeDescKind.ANY},
                 {25, 8, 25, 10, TypeDescKind.FLOAT},
-                {26, 20, 26, 23, TypeDescKind.ANY},
+                {26, 20, 26, 23, TypeDescKind.SINGLETON},
```

Comment:
@pubudu91 I changed this because inside the match clause, the type is narrowed to the matchExpr type